setup.py

```diff
@@ -75,7 +75,7 @@ def find_version(package):
         'OpenCL': ["pyopencl"],
         'CUDA': ["pycuda"],
     },
-    build_requires=['setuptools'],
-    test_requires=['pytest'],
-    cmdclass = {'test': PyTest},
+    #setup_requires=['setuptools'],
```

Comment:
The reason you removed `setup_requires` is that `setuptools` is being imported by this script anyway and we wouldn't be able to run it without `setuptools` installed, I guess?

Reply:
I removed it due to [this report](https://github.com/pytest-dev/pytest-xdist/issues/136) that pip doesn't work with `setup_requires`.  I haven't verified that this is an issue either way for sasmodels, so I'm going with less is more.